views/biological_information.py

```diff
@@ -19,9 +31,41 @@ def get_object(self):
         """
 
         person = self.request.person
+
         try:
             return person.biologicalinformation
         except AttributeError:
             raise NotFound(
                 detail='Associated biological information not found.'
```

Comment:
Add a trailing comma. Google why one should consider adding it. Or if you don't find an apt reason, @gauransh7 can elaborate.